src/leech/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Turn a story on certain websites into an ebook for convenient reading."""
+
+__version__ = "2.1"
```

Comment:
this version specifier is required here by flit, though it's redundant with the one in the main `leech.py` module.

Options would be to remove the other one, *or* move all of `leech.py` in to this `__init__.py`, so as to avoid the `leech.leech` redundancy.

Reply:
I noticed the sub-packages have code in their init files, so I went with that approach for the main package here.